controller/Cargo.toml

```diff
@@ -1,13 +1,12 @@
 [package]
 name = "epic_wallet_controller"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Developers <mimblewimble@lists.launchpad.net>"]
 description = "Controllers for epic wallet instantiation"
 license = "Apache-2.0"
 repository = "https://github.com/mimblewimble/epic-wallet"
```

Comment:
Change this repository to "https://github.com/EpicCash/epic-wallet"

Reply:
Done